pkgs/development/python-modules/gcal-sync/default.nix

```
@@ -27,6 +28,15 @@ buildPythonPackage rec {
     hash = "sha256-8VUXW6tIX43TV7UIxeforZIxAUqGY9uqpz6WGyH4d8E=";
   };
 
+  patches = [
+    (fetchpatch2 {
```

```suggestion
    (fetchpatch {
```
is preferred because it doesn't have problems with stable commit hashes.